00-programming-fundamentals/02-osx-useful-apps.md

```diff
@@ -0,0 +1,33 @@
+
+
+# OSX & Chrome Useful Apps
+
+## OSX Apps
+
+Below are some useful OSX apps for productivity as a developer.
+
+| App Name  | Description  |
+|---|---|
+| [Alfred](http://www.alfredapp.com/)  | Alfred is a replacement for Spotlight as an application launcher or automation tool.    |
+| [Slack](https://slack.com/downloads/osx)  | Slack has a desktop client for Mac Users.  |
```

Comment:
Yeah I thought about that, the only app we really require is Atom which we handle in installfest.  Maybe add Slack to it?